src/isIsogram.test.js

```diff
@@ -6,4 +6,29 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true if string has no repeating letters, consecutive or non-consecutivebe`, () => {
```

Comment:
There is a typo in the test description. The word 'non-consecutivebe' should be 'non-consecutive'.